_schemaV2.graphql

```diff
@@ -22198,15 +22153,21 @@ type Viewer {
     # Recaptcha token.
     recaptchaToken: String!
   ): VerifyUser
+
+  # Find a viewing room by ID
+  viewingRoom(id: ID!): ViewingRoom
```

Comment:
maybe I should remove this field since it was not there before

Reply:
Seems useful!